test/cpp_tests_list_IECPU.txt

```diff
@@ -7,92 +7,11 @@
 ####  EXCLUDED GTEST PATTERNS ####
 [EXCLUDED]
 
-BackendManager.* # backend tests don't apply for IE
-NGraphExecTest.Axpy
 NGraphExecTest.Axpy8bit
 NGraphExecTest.MixedTensors
-NGraphExecTest.FindNumberOfNodesUtil1
-NGraphExecTest.FindNumberOfNodesUtil2
-TFExec.SingleGraphOn2Threads
-TFExec.axpy
-tf_exec.BatchMatMul
-EncapsulateOp.GetNgExecutable
+BackendManager.SetBackend
+BackendManager.GetCurrentlySetBackendName
+BackendManager.CanCreateBackend
 OpByOpCapability.Backend
-MathOps.Sum
-MathOpsSumFixture.FullSet
-MathOps.Mean
-MathOps.ArgMaxNeg
```

Comment:
How are ArgMax, ArgMin, Atan2, BatchMatmul working with IE, these aren't upgraded to opset3?

Reply:
These tests + CF-pass are creating C->R, so they are not actually going to IE

Reply:
So there may be other tests doing the same thing but we actually want those to be tested on IE

Reply:
IE seemingly supports some pre-opset3 ops using that same path.

Reply:
Disabled the constant-folding pass for now.